CensusAnalytics/OpenDataLoader/ODL.Common/IngestBase.cs

```diff
@@ -7,10 +7,11 @@
 
 namespace ODL.Common
 {
-    internal abstract class IngestBase
+    public abstract class IngestBase

```

Comment:
I did this so my command line program would work. My understanding is that I can't access this or implementing classes unless they are public.